README.md

```diff
@@ -281,6 +282,24 @@ puts response
 # => Button::Response()
 ```
 
+### Offers
+
+##### Get Offers
+
+```ruby
+require 'button'
+
+client = Button::Client.new('sk-XXX')
+
+response = client.offers.get_offers({
+    user_id: "some-user-id",
+    device_ids: ["123"]
```

Comment:
What kind of device ids get submitted here?

Reply:
Same as above ^^